app/src/main/res/values-tr/strings.xml

```diff
@@ -411,61 +412,84 @@
-    <string name="reporter">Muhabir</string>
-    <string name="resolved_by">Çözen</string>
-    <string name="unresolve">Çöz</string>
-    <string name="unresolved_by">Çözümü Yapmayan</string>
-    <string name="reason">Nedeni</string>
-    <string name="creator">Yaratıcı</string>
+    <string name="reporter">Raporlayan</string>
+    <string name="resolved_by">Çözümleyen</string>
+    <string name="unresolve">Çözümü kaldır</string>
+    <string name="unresolved_by">Çözümü kaldıran</string>
+    <string name="reason">Neden</string>
+    <string name="creator">Oluşturan</string>
     <string name="message">Mesaj</string>
-    <string name="report_person">%1$s adlı kişiyi bildir</string>
-    <string name="block_person">%1$s adlı kişiyi engelle</string>
-    <string name="undistinguish_comment">Yorumları ayırt etme</string>
-    <string name="distinguish_comment">Yorumu ayırt et</string>
-    <string name="site_info_name">%1$s bilgi</string>
-    <string name="site_legal_info_name">%1$s yasal bilgi</string>
+    <string name="report_person">%1$s\'i raporla</string>
+    <string name="block_person">%1$s\'i engelle</string>
+    <string name="undistinguish_comment">Yorumu sıradanlaştır</string>
+    <string name="distinguish_comment">Yorumu öne çıkar</string>
+    <string name="site_info_name">%1$s bilgisi</string>
+    <string name="site_legal_info_name">%1$s yasal bilgisi</string>
     <string name="moderators">Moderatörler</string>
     <string name="admins">Yöneticiler</string>
-</resources>
\ No newline at end of file
+    <string name="custom_thumbnail">Özel Küçük Resim</string>
+    <string name="alt_text">Alternatif Metin</string>
+    <string name="post_hidden">Gönderi gizlendi</string>
+    <string name="post_unhidden">Gönderi gösterildi</string>
+    <string name="hide_post">Gönderiyi Gizle</string>
+    <string name="unhide_post">Gönderiyi Göster</string>
+    <string name="show_upvotes">Yukarı Oyları Göster</string>
+    <string name="show_downvotes">Aşağı Oyları Göster</string>
+    <string name="show_upvote_percentage">Yukarı Oy Yüzdesini Göster</string>
+    <string name="score">Puan</string>
+    <string name="info">Bilgi</string>
+    <string name="settings_about_baseline_profile">Temel profil</string>
+    <string name="installed">Yüklendi</string>
+    <string name="not_installed">Yüklenmedi</string>
+    <string name="unable_to_retrieve">Alınamıyor</string>
+    <string name="blocks">Engellemeler</string>
+    <string name="unblock">Engeli kaldır</string>
+    <string name="blocked_users">Engellenen kullanıcılar</string>
+    <string name="blocked_communities">Engellenen topluluklar</string>
+    <string name="blocked_instances">Engellenen örnekler</string>
+    <string name="you_have_no_blocked_users">Engellenen kullanıcınız yok</string>
+    <string name="you_have_no_blocked_communities">Engellenen topluluğunuz yok</string>
+    <string name="you_have_no_blocked_instances">Engellenen örneğiniz yok</string>
+<resources>
```

Comment:
Its `</resources>`

Reply:
Isn't that how it should be anyway? End code

Reply:
? you have a opening tag, you need a closing tag.

Reply:
> ? you have a opening tag, you need a closing tag.

I forgot the small detail :)